### DISCUSSION TITLE
AnnotatedSocketException

### DISCUSSION THREAD

**Discussion by mariussafelog** [eclipse-milo/milo#1204] [General]
I have the following Problem. I have a Milo client running which connects to a VM (1.2.3.4:4840) which redirects traffic to an Ocpua Server (5.6.7.8:4840). On my connect I get an exception that the network is not reachable, because the subnet of the server is not in reach of my client.
Is this because the client tries to answer to the IP contained in some handshake package? If so, is there an option to always answer to the configured address?

Thanks for your help!

java.util.concurrent.ExecutionException: io.netty.channel.AbstractChannel$AnnotatedSocketException: Network is unreachable: /5.6.7.8:4840
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999) ~[?:?]

**Comment by kevinherron** [comment]
See the advice in https://stackoverflow.com/a/40555243/85927
